env.example

```diff
@@ -77,6 +77,10 @@ JVB_STUN_SERVERS=stun.l.google.com:19302,stun1.l.google.com:19302,stun2.l.google
 # Media port for the Jitsi Videobridge
 JVB_PORT=10000
 
+# A comma separated list of APIs to enable when the JVB is started. The default is none.
+# See https://github.com/jitsi/jitsi-videobridge/blob/master/doc/rest.md for more information
+#JVB_APIS_TO_ENABLE=--apis=rest,colibri
```

Comment:
I don't want the user to specify the `--apis=` part, they should just say `none` or `rest,colibri`.